src/manifolds/GrassmannProjector.jl

```diff
@@ -0,0 +1,91 @@
+@doc raw"""
+    ProjectorPoint <: AbstractManifoldPoint
+
+A type to represent points on a manifold that are orthogonal projectors.
+"""
+struct ProjectorPoint{T<:AbstractMatrix} <: AbstractManifoldPoint
+    value::T
+end
+
+@doc raw"""
+    ProjectorTVector <: TVector
+
+A type to represent tangent vectors to points on a manifold that are orthogonal projectors.
+"""
+struct ProjectorTVector{T<:AbstractMatrix} <: TVector
+    value::T
+end
+
+@manifold_element_forwards ProjectorPoint value
+@manifold_vector_forwards ProjectorTVector value
+
+@doc raw"""
+    check_point(::Grassmann{n,k}, p::ProjectorPoint; kwargs...)
+
+Check whether a orthogonal projector is a point from the [`Grassmann`](@ref)`(n,k)` manifold,
+i.e. the [`ProjectorPoint`](@ref) ``p ∈ \mathbb R^{n×n}`` has to fulfill
+``p^{\mathrm{T}} = p``, ``p^2=p``, and ``\operatorname{rank} p = k`.
+"""
+function check_point(M::Grassmann{n,k,𝔽}, p::ProjectorPoint; kwargs...) where {n,k,𝔽}
```

Comment:
Docstring suggests it only works for the real case but method signature allows complex Grassmann manifold.

Reply:
You‘re right, I was not careful enough there.